plugins/outputs/cloudwatch/aggregator_test.go

```diff
@@ -39,6 +39,8 @@ func TestAggregator_NoAggregationKeyFound(t *testing.T) {
 
 	assertNoMetricsInChan(t, metricChan)
 	close(shutdownChan)
+	// Cleanup
+	wg.Wait()
```

Comment:
I definitely recall trying to do this to fix the flaky tests but it didn't completely fix it. In practice, I think this is what we need either way so I'm fine leaving it - just calling out I remember that this didn't fix it when I tried.. unless I did it wrong.

Reply:
Well shoot. I appreciate the call out.